README.md

```diff
@@ -1 +1,327 @@
-# messaging-with-jms-queues
\ No newline at end of file
+# Messaging with JMS Queues
+
+This guide walks you through the process of messaging with jms queues using a message broker in Ballerina language.
+Java Message Service (JMS) is used for sending messages between two or more clients. JMS support two models, point-to-point model and publish/subscribe model. This example is based on point-to-point model where messages are routed to an individual consumer which maintains a queue of "incoming" messages. This messaging type is built on the concept of message queues, senders, and receivers. Each message is addressed to a specific queue, and the receiving clients extract messages from the queues established to hold their messages. In point-to-point model each message is guaranteed to be delivered, and consumed by one consumer in an asynchronous manner.
+
+## <a name="what-you-build"></a>  What you’ll Build
+To understanding how you can use JMS queues for messaging, let's consider a real-world use case of an online bookstore service using which a user can order books for home delivery. Once an order is placed, the service will add it to a JMS queue named "OrderQueue" if the order is valid. Hence, this bookstore service acts as the JMS message producer. An order delivery system, which acts as the JMS message consumer polls the "OrderQueue" and gets the order details whenever the queue becomes populated. The below diagram illustrates this use case clearly.
+
+
+
+![alt text](https://github.com/pranavan15/messaging-with-jms-queues/blob/master/images/JMSQueue.png)
+
+
+
+In this example WSO2 EI Message Broker has been used as the JMS broker. Ballerina JMS Connector is used to connect Ballerina 
+and JMS Message Broker. With this JMS Connector, Ballerina can act as both JMS Message Consumer and JMS Message 
```

Comment:
Using WSO2 EI's broker profiles is not suitable for this use case. Either we can use Ballerina broker for third part broker such as ActiveMQ. Can you please change the use case accordingly.

Reply:
Requested change has been addressed. Changed the MB from WSO2 EI's broker to Apache ActiveMQ

Reply:
Hi,
Requested change has been addressed. Changed the MB from WSO2 EI's broker to Apache ActiveMQ. Please review the changes.